target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -308,7 +308,7 @@ define Device/prpl_haze
 	DEVICE_DTS_CONFIG := config@hk09
 	SOC := ipq8072
 	DEVICE_PACKAGES := ath11k-firmware-qcn9074 ipq-wifi-prpl_haze kmod-ath11k-pci \
-		mkf2fs f2fsck kmod-fs-f2fs kmod-leds-lp5562
+		kmod-fs-f2fs f2fs-tools kmod-leds-lp5562
```

Comment:
I miss description/reasoning about  this change in the commit. If its unrelated, move it into separate commit with proper description.

Reply:
description/reasoning: mkf2fs, f2fsck are part of metapackage f2fs-tools. f2fs-tools will be extended/adapted as f2fs devels see fit, instead of stagnating in the past. including the metapackage is the standard practice on other devices in openwrt, AFAICT.

> If its unrelated, move it into separate commit with proper description

i fixed [the device i own](https://github.com/openwrt/openwrt/commit/3b221ba5afe42bc32e878299d7d2275702536684). i don't have this device, i just wanted to help when i identified issues in other devices. you are more than welcome to take over these changes if you favor particular styles. in any case, thanks for the comment.